src/main/java/me/gaagjescraft/network/team/manhunt/games/Game.java

```diff
@@ -586,9 +619,7 @@ public void delete() {
     }
 
     public void create() {
-        int random = ThreadLocalRandom.current().nextInt(plugin.getCfg().seeds.size());
-        long seed = plugin.getCfg().seeds.get(random);
-        this.seed = seed;
+        this.seed = ThreadLocalRandom.current().nextLong();
```

Comment:
This removes the seed pre-selection system

Reply:
I guess you are looking at one particular commit. It was later reverted.